airbyte-integrations/connectors/source-woocommerce/source_woocommerce/manifest.yaml

```diff
@@ -156,6 +156,7 @@ definitions:
       end_time_option:
         field_name: before
         inject_into: request_parameter
+      cursor_field: "date_created_gmt"
```

Comment:
was this just not used?

Reply:
It is used in the manifest, which was why it triggered a CAT failure, but it was a mistake on my end that I accidentally deleted `date_created_gmt` cursor. I thought product reviews stream was using the same cursor as the other streams (`date_modified_gmt`) so it didn't need an override, but it wasn't so thats why I added this back